vms/platformvm/block/executor/verifier_test.go

```diff
@@ -1365,7 +1365,6 @@ func TestDeactivateLowBalanceL1ValidatorBlockChanges(t *testing.T) {
 			currentFork:       upgradetest.Etna,
 			durationToAdvance: time.Second,
 			networkID:         constants.UnitTestID,
-			expectedErr:       ErrStandardBlockWithoutChanges,
```

Comment:
The code now allows these block to be issued pre-Fortuna (as they should have been originally). This isn't a breaking change to the network because no networks will verify new (blocks that weren't already accepted) pre-Fortuna blocks.